db/forward_iterator.cc

```diff
@@ -75,6 +75,7 @@ class ForwardLevelIterator : public InternalIterator {
         read_options_, *(cfd_->soptions()), cfd_->internal_comparator(),
         files_[file_index_]->fd,
         read_options_.ignore_range_deletions ? nullptr : &range_del_agg,
+        cfd_->GetLatestMutableCFOptions()->prefix_extractor.get(),
```

Comment:
You can do that, but I think it's easier to find it in `sv_`.

Reply:
`ForwardIterator` has access to `sv_`, I'm passing it to `ForwardLevelIterator`'s constructor because I don't want to change all the function interfaces, the downside is that during the life cycle of `ForwardLevelIterator` the `prefix_extractor` won't change but I guess that's ok